### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by vmashnitskaya**

### CODE REVIEW

File: app/src/redux/data/dataSlice.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+export const fetchSortedFilteredSearchedMovies = createAsyncThunk<
+    Array<FilmData>,
+    {
+        searchValue: string;
+        filterValue: string;
+        sortData: SortData;
+    }
+>('data/fetchCustomData', async (args) => {
+    const { searchValue, filterValue, sortData } = args;
+    let url = 'http://localhost:4000/movies?';
```
Comment (GGWebProject) [inline app/src/redux/data/dataSlice.ts:46]: U can store url like const in your app

File: app/src/redux/data/dataSlice.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+            filterValue,
+            sortData,
+        })
+    );
+};
+
+export const deleteMovie = createAsyncThunk<void, number, { state: any }>(
+    'data/deleteMovie',
+    async (id, { getState, dispatch }) => {
+        const url = `http://localhost:4000/movies/${id}`;
```
Comment (GGWebProject) [inline app/src/redux/data/dataSlice.ts:77]: the same https://github.com/vmashnitskaya/react-mentoring-program-tasks/pull/5/files#r603335924

File: app/src/components/HomePage/ResultArea/MoreDropdown/MoreDropdown.tsx [hunk 3]
```diff
@@ -189,6 +181,7 @@ const MoreDropdown: FunctionComponent<MoreDropdownProps> = ({
                     toggleModalClose={toggleEditModal}
                     newMovieData={newMovieData}
                     onValueChanged={onValueChanged}
```
Comment (GGWebProject) [inline app/src/components/HomePage/ResultArea/MoreDropdown/MoreDropdown.tsx:183]: pls take a look at the [doc](https://reactjs.org/docs/faq-functions.html#how-do-i-pass-an-event-handler-like-onclick-to-a-component): your prop should be named `onValueChange`, for example.

File: app/.eslintrc.json [hunk 4]
```diff
@@ -25,6 +25,7 @@
     "camelcase": "off",
     "react/destructuring-assignment": "off",
     "jsx-a11y/no-noninteractive-tabindex": "off",
-    "jsx-a11y/no-noninteractive-element-interactions": "off"
+    "jsx-a11y/no-noninteractive-element-interactions": "off",
+    "no-param-reassign": "off"
```
Comment (GGWebProject) [inline app/.eslintrc.json:29]: It's not great. 
can u get rid of this?

**Comment by vmashnitskaya** [comment]
@GGWebProject , thank you for comments, will be fixed in further pr

**Pull request merged**